CONTRIBUTING.md

```diff
@@ -0,0 +1,70 @@
+# Hosting a Bitcoin Core review club meeting
+
+## General Tips
+
+Some tips for hosting a review club meeting:
+
+- try to post notes and questions on the Friday the week before the meeting.
+  That gives attendees time to review the PR and prepare for the meeting.
+- prepare a range of questions from basic to advanced. The review club is a way
+  to help new developers develop. There should be something for everone.
+- come with the mindset that the attendees are people who are as smart as you,
+  but don't have the context that you do in Bitcoin protocol development.
+- very open questions to the group (eg "Does everyone know how tx gossip works
+  in Bitcoin?") sometimes fall flat because all the attendees are waiting for
+  someone else to answer. Instead try to make the questions focussed on the
+  PR or change set that is being reviewed (eg "how does commit X change the
+  way bitcoind gossips transactions?")
+- start the meeting with a #startmeeting and 'hi' from everyone so you know
+  who's at their keyboard and paying attention.
+- a quick poll at the start ("Did everyone get a chance to review the PR? How
+  about a quick y/n from everyone") establishes who will probably be
+  asking/answering the more interesting questions.
+- don't be upset if people haven't had a chance to review the PR yet. This is a
+  voluntary meeting and most people have a lot of other commitments.
+- when hosting, it sometimes feels like there are minutes of silence and no-one
+  is out there. Be patient and don't worry! It takes people a bit of time to
+  formulate their thoughts and type them out.
+- it's nice to remind people of some of our meeting conventions at the start:
+  - you don't have to ask to ask a question (eg "I have a question about x but
+    I don't know if it's on-topic?"). Just go ahead and ask. If it's off-topic
+    we'll tell you.
+  - the host is there to help moderate, not to lead. You don't need to wait
+    for the host to ask a specific question - just jump in at any point.
+- keep an eye on the clock, and try to wrap up the meeting at the end of the
+  hour. Even if you're happy to continue beyond the hour, some of the attendees
+  might have hard stops.
+- be encouraging! For many people, asking questions can be intimidating, even
+  in the pseudonymous comfort of IRC! People are there to learn. Try to create
+  an environment where they feel safe to ask any questions and where they can
+  try to answer.
+- have fun, and pat yourself on the back for making Bitcoin protocol development
+  stronger and more decentralized 🚀
+
```

Comment:
If the PR is  one part of a serie, it's really interesting to do a bit of archeology and provide pointers to previous ones, especially if they're spanned on multiple years